File: 06Adapter/Students/2018/ZwaraPiotr/Adapter/Adapter.Zad1/Implementation/Aircraft.cs
using System;

using Adapter.Zad1.Abstraction;

namespace Adapter.Zad1.Implementation
{
    public sealed class Aircraft : IAircraft
    {
        public Aircraft()
        {
            this.Airborne = false;
            this.Height = 0;
        }

        public bool Airborne { get; private set; }

        public int Height { get; set; } // private set; }

        public void TakeOff()
        {
            Console.WriteLine("Aircraft engine takeoff");

            this.Airborne = true;
            this.Height = 200;
        }
    }
}

File: 06Adapter/Students/2019/SebastianPiekarek/src/ZAD1/Aircraft.java
package ZAD1;

public class Aircraft implements IAircraft {

    int height;
    boolean airborne;

    public Aircraft(){
        this.height = 0;
        this.airborne = false;
    }


    @Override
    public void takeOff() {
        System.out.println("Aircraft engine takeoff");
        airborne = true;
        height = 200;
    }

    public boolean getAirborne() {
        return airborne;
    }

    public int getHeight() {
        return height;
    }
}
